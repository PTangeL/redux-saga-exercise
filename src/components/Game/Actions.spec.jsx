import React from 'react'
import { shallow, mount } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Actions from './Actions'

describe('<Actions />', () => {
  const props = {
    like: () => {},
    dislike: () => {}
  }

  const wrapper = shallow(
    <Actions {...props} />
  )

  it('should render with props', () => {
      expect(shallowToJson(wrapper)).toMatchSnapshot()
    })

  it('Method #componentDidMount() should call likeButton.focus()', () => {
    //TODO implement test
  })

})
