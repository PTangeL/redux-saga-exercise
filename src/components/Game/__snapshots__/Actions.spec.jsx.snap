// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Actions /> should render with props 1`] = `
<div
  className="container"
  style={
    Object {
      "background": "white",
      "borderRadius": "0 0 25px 25px",
    }
  }
>
  <ul
    className="list-unstyled list-inline text-center game-actions"
  >
    <li>
      <button
        className="btn btn-danger btn-circle btn-xl"
        name="dislikeBtn"
        onClick={[Function]}
        type="button"
      >
        <i
          aria-hidden="true"
          className="fa fa-times"
        />
      </button>
    </li>
    <li />
    <li>
      <button
        className="btn btn-success btn-circle btn-xl"
        name="likeBtn"
        onClick={[Function]}
        type="button"
      >
        <i
          aria-hidden="true"
          className="fa fa-heart"
        />
      </button>
    </li>
  </ul>
</div>
`;
